src/tree/EnvironmentTreeItem.ts

```diff
@@ -174,6 +175,8 @@ export class EnvironmentTreeItem extends AzureParentTreeItem implements IAzureRe
         const branch: string | undefined = remote ? await tryGetLocalBranch() : undefined;
         this.inWorkspace = this.parent.repositoryUrl === remote && this.branch === branch;
 
-        this.gitHubConfigGroupTreeItems = await GitHubConfigGroupTreeItem.createGitHubConfigGroupTreeItems(this);
+        if (getWorkspaceSetting(enableLocalProjectView)) {
```

Comment:
The type for `public gitHubConfigGroupTreeItems: GitHubConfigGroupTreeItem[];` should include `| undefined` to make sure there's no accidental null ref errors